test/unittest/configuration/ConfigurationTest.cpp

```diff
@@ -213,6 +213,19 @@ std::string random_participant_name(
     return std::string("PartName_") + std::to_string(seed);
 }
 
+/*
+ * Random participant type
+ *
+ * WARNING: the max_types_available must be updated with each new type added
```

Comment:
Maybe in the future make ParticipantType::participant_type_with_name_ public so it is accessible from here.

Reply:
:raccoon: